website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -20,13 +20,21 @@ To add an application to authentik and have it display on users' **My applicatio
 
 2. Click **Create with Wizard**. (Alternatively, use our legacy process and click **Create**. The legacy process requires that the application and its authentication provider be configured separately.)
 
-3. In the **New application** wizard, define the application details, the provider type and configuration, and then click **Submit**.
+3. In the **New application** wizard, define the application details, the provider type, bindings for the application.
 
-4. To manage the display of the new application on the **My applications** page, you can optionally define the bindings for a specific policy, group, or user. Note that if you do not define bindings, then all users have access to the application, For more information, refer to [authorization](#authorization).
+    - **Application**: provide a name, an optional group for the type of application, the policy engine mode, and optional UI settings.
 
-## Authorization
+    - **Choose a Provider**: Select the provider types for this application.
 
-Application access can be configured using (Policy) bindings. Click on an application in the applications list, and select the _Policy / Group / User Bindings_ tab. There you can bind users/groups/policies to grant them access. When nothing is bound, everyone has access. You can use this to grant access to one or multiple users/groups, or dynamically give access using policies.
+    - **Configure a Provider**: Provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and any additional required configurations.
+
+    - **Configure Bindings**: To manage the display of the new application on the **My applications** page, you can optionally create a [binding](../flows-stages/bindings/index.md) between the application and a specific policy, group, or user. Note that if you do not define any bindings, then all users have access to the application. For more information about user access, refer to our documentation about [authorization](#policy-driven-authorization) and [hiding an application](#hide-applications).
+
+4. On the **Review and Submit Application** panel, review the configuration for the new application and its provider, and then click **Submit**.
+
+## Policy-driven authorization
+
+To use a [policy](../../customize/policies/index.md) to control which users or groups can access an application, click on an application in the applications list, and select the **Policy/Group/User Bindings** tab. There you can bind users/groups/policies to grant them access. When nothing is bound, everyone has access. Binding a policy restricts access to specific Users or Groups, or by other custom policies such as restriction to a set time-of-day or a geographic region.
```

Comment:
Nit: No comma needed in "... click on an application in the applications list and select..."  It's a two-step, the Oxford comma is dead-horse levels of overkill here.

Reply:
Ahah, you made me miss something that my BMC Editor would have had fits about.. for final step one has to say ", and then click..."